src/LanguageServer/Impl/Indexing/IndexManager.cs

```diff
@@ -57,18 +57,19 @@ internal class IndexManager : IIndexManager {
             _disposables
                 .Add(_symbolIndex)
                 .Add(() => idleTimeService.Idle -= OnIdle);
-
-            StartAddRootDir();
         }
 
         public int ReIndexingDelay { get; set; } = DefaultReIndexDelay;
 
-        private void StartAddRootDir() {
-            foreach (var fileInfo in WorkspaceFiles()) {
-                if (ModulePath.IsPythonSourceFile(fileInfo.FullName)) {
-                    _symbolIndex.Parse(fileInfo.FullName);
+        public Task IndexWorkspace(CancellationToken ct = default) {
```

Comment:
You're adding a cancellation token, but I don't see that it's actually used anywhere; it's always `default`. Is there any reason to add it if nobody cancels it?

Reply:
Even if it's not used, I'd have it there in case someone moves the call to IndexWorkspace